src/CommunityToolkit.Aspire.Client.MassTransit.RabbitMQ/CommunityToolkit.Aspire.Client.MassTransit.RabbitMQ.csproj

```
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+      <AdditionalPackageTags>Client Aspire MassTransit RabbitMQ</AdditionalPackageTags>
```

```suggestion
      <AdditionalPackageTags>Client MassTransit RabbitMQ</AdditionalPackageTags>
```

Aspire isn't required as a custom tag